runA11yTests.js

```diff
@@ -17,6 +17,34 @@ function resetReportDir( config ) {
 	fs.mkdirSync( config.reportDir, { recursive: true } );
 }
 
+// FIXME: Temporary pa11y workaround for color
+// contrast errors in two groups – first valid, second
+// invalid. Not suitable for production.
+// Switch to axe-core for better accuracy.
+function extractColorContrastInstances( data ) {
+	const colorContrastInstances = [];
+	let addingToArray = false;
+
+	for ( let i = 0; i < data.issues.length; i++ ) {
```

Comment:
[Suggestion] I'm wondering if you could use data.issues.filter.map here -t might be more readable?

const colorContrastIssues = data.issues.filter((issue)=> issue.code === 'color-contrast')